components/dashboard/src/settings/settings-menu.ts

```diff
@@ -1,11 +1,23 @@
 export default [
+    {
+        title: 'Account',
+        link: '/account'
+    },
+    {
+        title: 'Notifications',
+        link: '/notifications'
+    },
+    {
+        title: 'Plans',
+        link: '/plans'
+    },
     {
         title: 'Environment Variables',
-        link: '/env-vars'
+        link: '/environment-variables'
     },
     {
-        title: 'Git Integration',
-        link: '/git-integration'
+        title: 'Git Integrations',
+        link: '/git-integrations'
```

Comment:
**thought**: The initial idea was to have shorter menu items to avoid making the user interface feel crumped as more explicit information can be part of the subheading. For example, we could go with a) _Variables_ instead of _Environment Variables_ and b) _Integrations_ instead of _Git Integrations_. If so, the same pattern could be used also in URLs like `/integrations` instead of `/git-integrations`. WDYT?

However, I understand that we could be more explicit about the menu labels. 🤷

Reply:
Ah wait, I think I wrongly understood this:

> Account = Profile
> Plans = Subscriptions
> Variables = Environment Variables
> Integrations = Git Integrations
> Feature Preview = GONE :slightly_smiling_face:

We want the left ones, not the right ones, correct?

Also I don't like dashes in URLs (unclean/clutter), so single word URI segments sound good to me. Will adjust.

Reply:
Done 🙌